helm/thingsboard/templates/js-executor.yaml

```diff
@@ -47,6 +47,12 @@ spec:
       serviceAccountName: {{ include "thingsboard.serviceAccountName" . }}
       securityContext:
         {{- toYaml .Values.jsexecutor.podSecurityContext | nindent 8 }}
+      initContainers:
```

Comment:
Shall we add e2e tests to this repo as well to ensure they still green?

Reply:
> > > Is this change necessary because the pods don't check for its dependencies before marking them as ready? if the pods fail initially, aren't they marked as ready/live eventually when the dependencies are alive?
> 
> > I think they check it, but doing so means to start the pod and that takes more resources than the init containers. So this should make the startup of TB more quick as there are less pods fighting for resources
> 
> But this is due to our testing environment being resource constrained, that shouldn't be a problem in a production setup right? I guess my concern is that we're leaking environment limitations into something that should be environment agnostic and that raises a flag in my head that something is off somewhere.

This should not be an issue in prod environments because they are no so resource constrained but this will be also helpful for them to start faster. Notice that this change mainly applies for new deployments because for other scenarios  databases will be already up and ready!